v2/cn/ann-1019009001rc1.md.tt2

```diff
@@ -0,0 +1,108 @@
+    * change: it is harmful to set ngx.ctx to a non-table value. _Thanks spacewander for the patch._
+* upgraded [lua-tablepool](https://github.com/openresty/lua-tablepool) to v0.02
+    * bugfix: should also clear the metatable when clearing key values of a table. _Thanks doujiang24 for the patch._
+    * change: discard the object simply when the pool size is large than `max_pool_size`, so that we can get better performance. _Thanks doujiang24 for the patch._
+* upgraded [lua-resty-memcached](https://github.com/openresty/lua-resty-memcached) to v0.16rc1
+    * change: use preferred settimeouts instead of settimout. _Thanks Elvin Efendi for the patch._
+* upgraded [lua-resty-dns](https://github.com/openresty/lua-resty-dns) to v0.22rc1
+    * feature: add `no_random` option. _Thanks Thijs Schreijer for the patch._
+* upgraded [lua-resty-mysql](https://github.com/openresty/lua-resty-mysql) to v0.24rc1
+    * bugfix: 64-bit integer values in the MySQL packets (like last `insert_id`) could not be properly parsed due to LuaJIT's BitOp functions return results in the range of signed 32 bit numbers. _Thanks doujiang24 for the patch._
+* upgraded [opm](https://github.com/openresty/opm) to v0.0.6rc1
+    * feature: web: support displaying and searching package docs. _Thanks xlibor for the patch._
+    * change: print usage if no command specified. _Thanks tison for the patch._
+    * feature: web: improvements for the website UI. _Thanks xlibor for the patch._
+* upgraded [resty-cli](https://github.com/openresty/resty-cli) to v0.28rc1
+    * bugfix: share dict name can be upper letter & numbers. _Thanks Zhenzhen Zhao for the patch._
+    * bugfix: resty: win32: we should search for `*.ljbc` files first too.
+* upgraded [lua-resty-lrucache](https://github.com/openresty/lua-resty-lrucache) to v0.11rc1
+    * bugfix: set `num_items` to zero when `flushall`. _Thanks hnlq715 for the patch._
+* upgraded [lua-resty-string](https://github.com/openresty/lua-resty-string) to v0.14rc1
+    * feature: added parameter check for `aes.lua`. _Thanks lijunlong for the patch._
+    * feature: added aes gcm encryption/decryption algorithm. _Thanks lijunlong for the patch._
+    * bugfix: segmentation fault while decoding because the wrong output size buffer.  _Thanks Miroslav Nový for the patch._
+* upgraded [lua-resty-signal](https://github.com/openresty/lua-resty-signal) 0.03rc1
+    * change: the signum method return correct value on different system now, not only for linux. _Thanks xiaobiaozhao for the patch._
+* upgraded [LuaJIT](https://github.com/openresty/luajit2) to 2.1-20210510.
+    * bugfix: Fix trace exit register dump for some archs.
+    * bugfix: FFI: Fix recording of union initialization.
+    * bugfix: x64: Fix `__call` metamethod return dispatch.
+    * bugfix: Fix binary number literal parsing.
+    * feature: enable debuginfo in our luajit build by default.
+
```

Comment:
We should list all LuaJIT changes since the last OpenResty release. It cannot be just so few here. One of a very important one is the fixed order Lua table iteration mode. Why not mention it at all here?